src/main/resources/application.properties

```diff
@@ -5,4 +5,7 @@ spring.datasource.username=kore
 spring.datasource.password=start1
 
 logging.level.org.hibernate.SQL=DEBUG
-logging.level.org.hibernate.type.descriptor.sql.BasicBinder=TRACE
\ No newline at end of file
+logging.level.org.hibernate.type.descriptor.sql.BasicBinder=TRACE
+
+spring.security.user.name=boredoom
+spring.security.user.password=password123
```

Comment:
Am besten lagern wir die Credentials nochmal in eine lokale Konfigurationsdatei aus

Reply:
Können wir aber ein separates Ticket für anlegen

Reply:
Ich habe ein Ticket angelegt